ultraLatency.py

```diff
@@ -54,6 +54,9 @@
     print("Invalid MSPL specified: %d, must be in range [1,8]"%(options.MSPL))
     exit(1)
 
+if (options.stepSize + options.scanmin > options.scanmax):
+    options.stepSize = options.scanmax - options.scanmin
+
```

Comment:
You should not try to change options.stepSize as this is cmd line entry from the user and may be required elsewhere.

Instead you should store options.stepSize as a local variable and then modify this variable accordingly